src/tikzplotly/_scatter3d.py

```diff
@@ -0,0 +1,97 @@
+    marker = scatter.marker
+
+    options_dict = {}
+    mark_option_dict = {}
+
+    # Markers only
+    if mode == "markers":
+        if marker.symbol is not None:
+            symbol, symbol_options = marker_symbol_to_tex(marker.symbol)
+            options_dict["mark"] = symbol
+            options_dict["only marks"] = None
+            if symbol_options is not None:
+                mark_option_dict[symbol_options[0]] = symbol_options[1]
+        else:
+            options_dict["only marks"] = None
+
+        if marker.size is not None:
+            size = marker.size
+            if isinstance(size, (list, tuple)) or (hasattr(size, "shape") and hasattr(size, "__len__")):
+                try:
+                    import numpy as np
+                    size = float(np.mean(size))
+                except Exception:
+                    size = float(size[0])
+            options_dict["mark size"] = px_to_pt(size)
+
+        if marker.color is not None:
+            color_set.add(convert_color(marker.color)[:3])
+            mark_option_dict["solid"] = None
+            mark_option_dict["fill"] = convert_color(marker.color)[0]
+
+        if (line := marker.line) is not None:
+            if line.color is not None:
+                color_set.add(convert_color(line.color)[:3])
+                mark_option_dict["draw"] = convert_color(line.color)[0]
+            if line.width is not None:
+                mark_option_dict["line width"] = px_to_pt(line.width)
+
+        if (opacity := scatter.opacity) is not None:
+            options_dict["opacity"] = opacity
+        if (opacity := marker.opacity) is not None:
+            mark_option_dict["opacity"] = opacity
+
+        if mark_option_dict:
+            mark_options = option_dict_to_str(mark_option_dict)
+            options_dict["mark options"] = f"{{{mark_options}}}"
+
+    elif mode == "lines":
+        options_dict["mark"] = "none"
+
+    elif "lines" in mode and "markers" in mode:
+        if marker.symbol is not None:
+            symbol, symbol_options = marker_symbol_to_tex(marker.symbol)
+            options_dict["mark"] = symbol
+            if symbol_options is not None:
+                mark_option_dict[symbol_options[0]] = symbol_options[1]
+
+    else:
+        warn(f"Scatter3d : Mode {mode} is not supported yet.")
+
+    if scatter.line is not None:
+        if scatter.line.width is not None:
+            options_dict["line width"] = px_to_pt(scatter.line.width)
+        if scatter.line.color is not None:
+            options_dict["color"] = convert_color(scatter.line.color)[0]
+            color_set.add(convert_color(scatter.line.color)[:3])
+
+    if scatter.showlegend is False:
+        options_dict["forget plot"] = None
+
+    options = option_dict_to_str(options_dict)
+    if scatter.name:
+        code += f"\n% {scatter.name}\n"
+    
+    code += f"\\addplot3+ "
+    if options is not None:
+        code += f"[{options}]"
+    else:
+        code += "[only marks]"
```

Comment:
is there a particular reason for this option `only marks`? for a figure with `mode=markers+lines`, it adds this option while it shouldn't